src/pages/zh-cn/guides/deploy/gitlab.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: 部署你的 Astro 站点至 GitLab Pages
+description: 如何使用 GitLab Pages 将你的 Astro 网站部署到网络上。
+layout: ~/layouts/DeployGuideLayout.astro
+i18nReady: true
+---
+
+你可以使用 [GitLab Pages](https://pages.gitlab.io/) 为你的 [GitLab](https://about.gitlab.com/) 项目、组或用户账户托管 Astro 网站。
+
+:::tip[正在寻找示例?]
+看一看 [GitLab Pages 官方的 Astro 示例项目](https://gitlab.com/pages/astro)!
+:::
+
+## 如何部署
+
+1. 在 `astro.config.mjs` 文件中设置正确的 `site`。
+2. 在 `astro.config.mjs` 文件中设置 `outDir:public`。这会告诉 Astro 将静态构建输出放在 `public` 目录下，该目录是 GitLab Pages 指定的静态资源存放位置。
+
+   如果你在 Astro 项目中使用 [`public/` 目录](/en/core-concepts/project-structure/#public) 存放静态资源，你需要重命名该目录，并在 `astro.config.mjs` 文件中将 `publicDir` 的值改为新的目录名。
```

Comment:
```suggestion
   如果你在 Astro 项目中使用 [`public/` 目录](/zh-cn/core-concepts/project-structure/#public) 存放静态资源，你需要重命名该目录，并在 `astro.config.mjs` 文件中将 `publicDir` 的值改为新的目录名。
```
Here it can be changed to zh-cn.

Reply:
I fixed it. Thanks for the reminder!